src/interceptors/create-bundle-information-error.interceptor.ts

```diff
@@ -0,0 +1,20 @@
+import { CallHandler, ExecutionContext, Injectable, NestInterceptor } from '@nestjs/common';
+import { map, Observable } from 'rxjs';
+
+@Injectable()
+export class CreateBundleInformationErrorInterceptor implements NestInterceptor {
+  intercept(context: ExecutionContext, next: CallHandler): Observable<any> {
+    return next.handle().pipe(
+      map((data) => {
+        const response = context.switchToHttp().getResponse();
+        if (!data.facilityIdentifier) {
```

Comment:
One of the endpoints PUT facilities/:facilityIdentifier has different response bodies depending on the query parameter and we don't want the interceptor to change those ones

Reply:
@francescastocco and I discussed this and - since we changed the structure of the data for the interceptor anyway - we were able to make this check more specific by checking if the `data` has `responseBody` and `warningErrors` fields